scripts/projects-test/utils.ts

```diff
@@ -6,9 +8,35 @@ import sh from '../gulp/sh';
 // (will not catch SIGINT on windows)
 tmp.setGracefulCleanup();
 
-export function createTempDir(prefix: string): string {
```

Comment:
you missed a usage of `createTempDir: https://github.com/microsoft/fluentui/blob/05c7b0b06974893b2a49842d51be5444fffa2670/scripts/projects-test/packPackages.ts#L9

Reply:
also why the rename ?

Reply:
The rename is to reflect that it does more now beyond just creating the one directory. I ended up bringing back the original basic `createTempDir` in addition to the new function.